### PR TITLE
Script for generating public database dump

### PR DESCRIPTION
**Pull request by mayaCostantini**
## Related Issues and Dependencies

Related to #32 

## This introduces a breaking change

- [x] No

## This Pull Request implements

Script to generate a public dump of the database removing sensitive information and solver entries for other environments than `rhel8:py38`

### CODE REVIEW

**Comment by mayaCostantini** [comment]
/hold
Needs testing

File: generate_dump.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+
+psql -h "$KNOWLEDGE_GRAPH_HOST" -p "$KNOWLEDGE_GRAPH_PORT" --username=postgres < "$DUMP_NAME"
+psql -d "$KNOWLEDGE_GRAPH_DATABASE" -h "$KNOWLEDGE_GRAPH_HOST" -p "$KNOWLEDGE_GRAPH_PORT" --U "$KNOWLEDGE_GRAPH_USER"
+
+psql -d postgres -h localhost -p 5432 -U postgres -c 'DROP TABLE IF EXISTS kebechet_github_installations'
+psql -d postgres -h localhost -p 5432 -U postgres -c 'UPDATE python_package_metadata SET author="deducted", maintainer="deducted", maintainer_email="deducted", author_email="deducted"'
+psql -d postgres -h localhost -p 5432 -U postgres -c 'DELETE FROM solved WHERE document_id ~ "^(?!solver-rhel-8-py38)"'
+
```
Comment (pacospace) [inline generate_dump.sh:10]: what about data in adviser documents that we sync from Kebechet?

File: generate_dump.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+
+psql -h "$KNOWLEDGE_GRAPH_HOST" -p "$KNOWLEDGE_GRAPH_PORT" --username=postgres < "$DUMP_NAME"
+psql -d "$KNOWLEDGE_GRAPH_DATABASE" -h "$KNOWLEDGE_GRAPH_HOST" -p "$KNOWLEDGE_GRAPH_PORT" --U "$KNOWLEDGE_GRAPH_USER"
+
+psql -d postgres -h localhost -p 5432 -U postgres -c 'DROP TABLE IF EXISTS kebechet_github_installations'
```
Comment (pacospace) [inline generate_dump.sh:7]: is this going to be a cascade operation removing all linked docs? See question below

**Review by fridex — APPROVED** [review]
looks really good to me 👍🏻

**Pull request merged**